sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/evaluate/ExecutorResourceMapper.java

```diff
@@ -39,7 +38,6 @@ public ExecutorResourceMapper(
         this.resourceSpecs = resourceSpecs;
         this.executorResources = executorResources;
         this.resourceNamespace = resourceNamespace;
-        this.useDefaultExecutor = useDefaultExecutor;
```

Comment:
Missed the `useDefaultExecutor` parameter in the constructor (at L35)

Reply:
fixed